pom.xml

```diff
@@ -195,6 +209,11 @@
             <artifactId>kafka-models</artifactId>
             <version>${kafka-models.version}</version>
         </dependency>
+        <dependency>
```

Comment:
i see you've installed this but haven't excluded it anywhere else. This could cause a conflict, no?